src/account/helpers.js

```diff
@@ -65,14 +65,17 @@ export function listTokenAccounts(account: Account): TokenAccount[] {
 }
 
 export function flattenAccounts(
-  topAccounts: Account[] | TokenAccount[] | (Account | TokenAccount)[]
+  topAccounts: Account[] | TokenAccount[] | (Account | TokenAccount)[],
+  enforceHideEmptyTokenAccounts?: boolean
```

Comment:
for readability i suggest this is a `{ enforceHideEmptyTokenAccounts?: boolean }`